### PR TITLE
Preserve original linebreaks in the instructions strings.

### PR DESCRIPTION
**Pull request by krithin**
Resolves https://github.com/findthemasks/findthemasks/issues/309

### CODE REVIEW

File: public/locations-list-map.js [hunk 1]
```diff
@@ -195,6 +201,17 @@ function ctn(text) {
   return document.createTextNode(text);
 }
 
+// Turns a string with embedded \n characters into an Array of text nodes separated by <br>
+function multilineStringToNodes(input) {
+  const textNodes = input.split('\n').map((s) => ctn(s));
```
Comment (drwestco) [inline public/locations-list-map.js]: Stupid jscript trick. You can eliminate the foreach down below and turn this into a one-liner:
```
return [].concat(...input.split('\n').map((s) => [ ctn(s), ce('br') ]));

File: public/locations-list-map.js [hunk 2]
```diff
@@ -195,6 +201,17 @@ function ctn(text) {
...
+  let returnedNodes = [];
+  textNodes.forEach((e) => {
+    returnedNodes.push(e);
+    returnedNodes.push(document.createElement('br'));
+  })
+  return returnedNodes.slice(0,-1);
```
Comment (drwestco) [inline public/locations-list-map.js]: You can just decrement the length of returnedNodes to chop off the last <br>, instead of making a copy via slice()

**Comment by krithin** [comment]
I'm afraid I wasn't able to get to this yesterday and won't today either,
so feel free to push commits to this branch if you'd like! And if not I
will be able to do something here tomorrow.

**Comment by awong-dev** [comment]
@drwestco  if you'd like to make the efficiency fixes, please do so in a follow-up. Merging this.

**Pull request merged**